src/DependencyInjection/LazyContainerFactory.php

```diff
@@ -425,6 +426,10 @@ public function create(): RectorConfig
         $rectorConfig->singleton(FileProcessor::class);
         $rectorConfig->singleton(PostFileProcessor::class);
 
+        $rectorConfig->when(PostFileProcessor::class)
```

Comment:
This is used to provide these additional post rectors to the post file processor